content/en/docs/concepts/services-networking/_index.md

```diff
@@ -3,8 +3,51 @@ title: "Services, Load Balancing, and Networking"
 weight: 60
 description: >
   Concepts and resources behind networking in Kubernetes.
+no_list: true
 ---
 
+## The Kubernetes network model
+
+Every [`Pod`](/docs/concepts/workloads/pods/) gets its own IP address. 
+This means you do not need to explicitly create links between `Pods` and you 
+almost never need to deal with mapping container ports to host ports.  
+This creates a clean, backwards-compatible model where `Pods` can be treated 
+much like VMs or physical hosts from the perspectives of port allocation, 
+naming, service discovery, load balancing, application configuration, 
```

Comment:
Link "load balancing" to https://kubernetes.io/docs/concepts/services-networking/ingress/#load-balancing